tasks/EGPSR.tex

```diff
@@ -19,19 +14,12 @@ \subsection*{Focus}
 % Setup
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection*{Setup}
+Generally assume that the arena is supposed to be in its default state. Therefore, possible problems to solve are:
```

Comment:
The way egpsr worded is quite strange in my opinion. It is made very clear what the task is but not how this task fits in a household. Essentially, this is a combination of cleanup and gpsr. The robot has to maintain a clean house and act as a smart speaker.

Reply:
Yes, agreed. I updated the wording of the objective and the setup. I hope it reads better now